src/inmanta/deploy/scheduler.py

```diff
@@ -631,6 +651,8 @@ async def get_resource_intent(self, resource: ResourceIdStr) -> Optional[Resourc
             return ResourceIntent(model_version=self._state.version, details=resource_details, dependencies=None)
 
     async def get_resource_intent_for_deploy(self, resource: ResourceIdStr) -> Optional[ResourceIntent]:
```

Comment:
I would rename this to deploy_start (unrelated to this ticket, now may not be the time, so eel free to ignore)